pyspectral/tests/test_rayleigh.py

```diff
@@ -115,40 +113,28 @@ def setUp(self):
 
             self.viirs_rayleigh = rayleigh.Rayleigh('NOAA-20', 'viirs', atmosphere='midlatitude summer')
 
-    @patch('os.path.exists')
-    @patch('pyspectral.utils.download_luts')
-    @patch('pyspectral.rayleigh.get_reflectance_lut_from_file')
-    @patch('pyspectral.rsr_reader.RelativeSpectralResponse.'
-           '_get_rsr_data_version')
-    @patch('pyspectral.rayleigh.Rayleigh.get_effective_wavelength')
-    def test_dask_cliping_angles_with_nans(self, get_effective_wvl,
-                                           get_rsr_version, get_reflectance_lut_from_file,
-                                           download_luts, exists):
-        """Test the cliping of angles outside coordinate range - with nan's in input."""
-        get_reflectance_lut_from_file.return_value = (self.rayl, self.wvl_coord, self.azid_coord,
-                                                      self.satz_sec_coord, self.sunz_sec_coord)
-        download_luts.return_code = None
-        exists.return_code = True
-        get_rsr_version.return_code = RSR_DATA_VERSION
-        get_effective_wvl.return_value = self.cwvl
-
+    def test_dask_cliping_angles_with_nans(self):
+        """Test the clipping of angles outside coordinate range - with nan's in input."""
+        from pyspectral.rayleigh import _clip_angles_inside_coordinate_range
         zenith_angle = da.array([79., 69., 32., np.nan])
-        result = self.viirs_rayleigh._clip_angles_inside_coordinate_range(zenith_angle, 2.75)
-
```

Comment:
This test (and its counterpart in the other class) are a little too bad. Full set of mocks and they were never actually used/needed.